continuous_integration/recipes/distributed/conda_build_config.yaml

```diff
@@ -1,3 +1,3 @@
-python:
-    - 3.8
-    - 3.9
```

Comment:
It looks like this is no longer building for Python 3.9. Only Python 3.8. So I think we need to add this back

<details>

```
INFO :: The inputs making up the hashes for the built packages are as follows:
{
  "distributed-2022.02.1a220222-py38_ge7da29f6_50_cython": {
    "recipe": {
      "c_compiler": "gcc",
      "cython_enabled": "True",
      "target_platform": "linux-64"
    }
  },
  "distributed-2022.02.1a220222-py_ge7da29f6_50_python": {
    "recipe": {
      "cython_enabled": "False"
    }
  },
  "distributed-impl-2022.02.1a220222-cython": {
    "recipe": {}
  },
  "distributed-impl-2022.02.1a220222-python": {
    "recipe": {}
  }
}
```

</details>